MonoGame.Framework/GameWindow.cs

```diff
@@ -34,13 +34,13 @@ public abstract class GameWindow
         /// </summary>
         public virtual bool AllowAltF4 { get { return _allowAltF4; } set { _allowAltF4 = value; } }
 
-#if (WINDOWS && !WINDOWS_UAP) || DESKTOPGL
```

Comment:
Thinking we should just expose this on all platforms and let it be ignored on platforms that cannot use it.

Reply:
Agreed.